iceoryx_posh/test/moduletests/test_mepoo_config.cpp

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2021 by  Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "iceoryx_posh/iceoryx_posh_types.hpp"
+#include "iceoryx_posh/mepoo/mepoo_config.hpp"
+#include "iceoryx_utils/internal/posix_wrapper/shared_memory_object/allocator.hpp"
+#include "test.hpp"
+
+using namespace ::testing;
+
+using namespace iox::mepoo;
+
+class MePooConfig_Test : public Test
```

Comment:
Would be nice to test the `Entry` struct in the MepooConfig if the members are correctly default initialized.

Reply:
@dkroenke I'm not sure if I understood it correctly. There is no default constructor for Entry right ? We need to always pass the value to initialize the entry object. So I don't know if its possible to set the members of Entry to zero by default even though in the definition there is a default value given to the members.

Reply:
Yep right, i oversaw that there is only the specialized ctor, then it's fine^^